suave/gen/suave_spec.yaml

```diff
@@ -445,3 +445,33 @@ functions:
         - name: value
           type: bytes
           description: "Randomly-generated bytes"
+  - name: secp256k1Encrypt
```

Comment:
Can we create a generic encrypt/decrypt pair of precompiles and put both AES and this implementation there?